app/src/main/kotlin/org/fossify/clock/activities/WidgetDigitalConfigureActivity.kt

```diff
@@ -84,6 +84,10 @@ class WidgetDigitalConfigureActivity : SimpleActivity() {
         }
 
         updateTextColor()
+
+        val clockFormat = if (config.use24HourFormat) "HH:mm" else "K:mm a"
```

Comment:
We should be using `h:mm a` as in other clock apps, not `k:mm a`. Both 12 AM and PM hours now show up as 0 because of this.